build.sbt

```diff
@@ -45,7 +45,7 @@ lazy val root = (project in file("."))
 lazy val macros = (project in file("macros"))
   .settings(
     name := "tagless-redux-macros",
-    libraryDependencies += "org.typelevel" %% "cats-tagless-macros" % taglessV,
+    libraryDependencies += "org.typelevel" %% "cats-tagless-macros" % taglessV % "test",
```

Comment:
intentional?

Reply:
Yes, I added that as I was poking around more as it seems like these only get used during test. (Perhaps I should've switched to a new branch)